EAO/UI/Inspections/ProjectListController.swift

```diff
@@ -23,35 +25,45 @@ final class ProjectListController: UIViewController {
     //MARK:-
     override func viewDidLoad() {
         searchBar.returnKeyType = .default
-        load()
+        refreshData()
+        style()
     }
 
+    func style(){
+        refreshControl.attributedTitle = NSAttributedString(string: "Pull to Refresh")
+        refreshControl.addTarget(self, action: #selector(ProjectListController.refreshData), for: UIControl.Event.valueChanged)
+        tableView.addSubview(refreshControl)
+    }
+    
 	//MARK: IB Actions
 	@IBAction func customTapped(_ sender: UIBarButtonItem) {
+        
 		let alert = UIAlertController(title: "Project name", message: nil, preferredStyle: .alert)
 		alert.addTextField { (textField) in
 			textField.placeholder = "Start Typing..."
 		}
 		let select = UIAlertAction(title: "Select", style: .default) { (_) in
 			if let text = alert.textFields?.first?.text, !text.isEmpty(){
 				self.result?(text)
-				self.pop()
+				self.popViewController()
```

Comment:
Unwind

Reply:
In this an all other cases below all the information is either sent back via callbacks or saved into realm before transitions. And since unwind segue is going to be called from code, I think that using it adds extra code and complexity without any benefits